docker/docker.env

```diff
@@ -7,9 +7,6 @@ FORCE_COLOR=2
 # `.gmrc` checks for interactive TTY.
 PG_DUMP=docker-compose exec -T db pg_dump
 
-# Instructs to use our docker PostgreSQL service.
-DATABASE_HOST=db
-ROOT_DATABASE_URL=postgres://postgres@db/template1
```

Comment:
I don't think we should remove that, because it's there to override the .env vars when using docker-compose